### PR TITLE
CSS

### PR DESCRIPTION
**Pull request by mariaalouisaa**
- Added styling that I hope should still be responsive when everything comes together!
- When writing the HTML in the `server.send( ) ` try remember to use the` <head></head>` contents below as that has the fonts and stylesheet linked
- I tried to make the class names as obvious as poss, with little comments about where to use them (which we can remove later)
- Will do an accessibility check when the page is further along (for colours contrast)

Hope you like it so far 🙂

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  max-width: 700px;
+  padding: 1em 1.5em;
+  margin: 0 auto;
+  background-color: rgb(252, 233, 210);
+}
+
+/* Two colours for the header */
+h1 {
+  font-family: "Nunito", sans-serif;
+  color: orangered;
```
Comment (Glombort) [inline public/style.css]: Lighthouse doesn't like this for contrast, it suggests #c43500 for AAA rating

File: public/style.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  font-size: 3.5em;
+}
+
+.green {
+  /* Include inside the h1 (in a span)*/
+  color: green;
```
Comment (Glombort) [inline public/style.css]: This passes AA rating on lighthouse but it suggests #007b00 for AAA rating

**Review by Glombort — APPROVED** [review]
Nice looks super great, loving the half colour scheme:)

**Review by AsmahanM — APPROVED** [review]

File: public/style.css (hunk 1 above)
Comment (mariaalouisaa) [reply public/style.css]: Sweet thanks for checking that for me. Changed those now 👍

**Review by AsmahanM — APPROVED** [review]

**Pull request merged**